src/test/java/uk/ac/ebi/ampt2d/test/configurationaccession/StudyAccessioningDatabaseServiceTestConfiguration.java

```diff
@@ -0,0 +1,46 @@
+/*
+ *
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.test.configurationaccession;
+
```

Comment:
Can it be uk.ac.ebi.ampt2d.accession.testconfiguration ?

Reply:
It should be only .test.configuration a rogue namechange probably.